docs/sources/reference/components/otelcol/otelcol.receiver.solace.md

```diff
@@ -0,0 +1,210 @@
+---
+canonical: https://grafana.com/docs/alloy/latest/reference/components/otelcol/otelcol.receiver.solace/
+description: Learn about otelcol.receiver.solace
+title: otelcol.receiver.solace
+---
+
+# otelcol.receiver.solace
+
+`otelcol.receiver.solace` accepts traces from a [Solace PubSub+ Event Broker](https://solace.com/products/event-broker/) and
+forwards it to other `otelcol.*` components.
+
+> **NOTE**: `otelcol.receiver.solace` is a wrapper over the upstream
+> OpenTelemetry Collector `solace` receiver from the `otelcol-contrib`
+> distribution. Bug reports or feature requests will be redirected to the
+> upstream repository, if necessary.
+
+Multiple `otelcol.receiver.solace` components can be specified by giving them
+different labels.
+
+## Usage
+
+```alloy
+otelcol.receiver.solace "LABEL" {
+  queue = "QUEUE"
+  auth {
+    // sasl_plain or sasl_xauth2 or sasl_external block
+  }
+  output {
+    traces  = [...]
+  }
+}
+```
+
+## Arguments
+
+The following arguments are supported:
+
+| Name                 | Type     | Description                                                               | Default          | Required |
+| -------------------- | -------- | ------------------------------------------------------------------------- | ---------------- | -------- |
+| `queue`              | `string` | Name of the Solace telemetry queue to get span trace messages from.       |                  | yes      |
+| `broker`             | `string` | Name of the Solace broker using amqp over tls.                            | `localhost:5671` | no       |
+| `max_unacknowledged` | `int`    | Maximum number of unacknowledged messages the Solace broker can transmit. | 10               | no       |
+
+`queue` should have the format `queue://#telemetry-myTelemetryProfile`.
```

Comment:
Should or must? Should implies it can have some other format too.

Reply:
"must" is better then

Reply:
I updated it